### ISSUE TITLE
inputStream in XMLDocumentTest doesn't specify encoding type

### ISSUE THREAD

**Issue by simonjenga** [jcabi/jcabi-xml#106]
```inputStream``` in ```com.jcabi.xml.XMLDocumentTest``` doesn't specify encoding type

```
com.jcabi.xml.XMLDocumentTest.findsDocumentNodesWithXpathAndReturnsThem();
```

We're not quite sure which encoding type these tests pass against or which they may fail against.
Please supply the encoding type for clarity.

**Comment by dmarkov** [comment]
@yegor256 take a look at this issue please and dispatch it somehow, see [par.21](http://at.teamed.io/policy.html#21)

**Comment by yegor256** [comment]
@simonjenga but the text is in plain English, no need to specify encoding there. any encoding will work.
